### ISSUE TITLE
np.array_split on pd.Series causing unavoidable deprecation warnings for Series.swapaxes

### ISSUE THREAD

**Issue by Rick-OConnor** [ing-bank/EntityMatchingModel#9]
As reported in [this issue](https://github.com/numpy/numpy/issues/24889) on from numpy, the use of np.array_split on a pandas object results in an unavoidable 
```
FutureWarning: 'Series.swapaxes' is deprecated and will be removed in a future version. Please use 'Series.transpose' instead.
```

[This reply](https://github.com/numpy/numpy/issues/24889#issuecomment-1895503977) in the issue thread suggest an alternative solution to np.array_split, relying on `iloc` instead.

**Comment by mbaak** [comment]
Thanks, I'll update this in the next patch release.

**Comment by mbaak** [comment]
Resolved with v2.1.0.
